src/corefx/System.Globalization.Native/localeNumberData.cpp

```diff
@@ -31,9 +47,259 @@ enum LocaleNumberData : int32_t
 
 /*
 Function:
+NormalizePattern
+
+Returns a numeric string pattern in a format that we can match against the appropriate managed pattern.
+*/
+UnicodeString* NormalizePattern(const UnicodeString *srcPattern, UnicodeString *destPattern, bool isNegative)
```

Comment:
No it's not required in this case. I followed the coding patterns present in ICU that takes a string by ref and then outputs that same string for callers that want to easily chain methods. (http://userguide.icu-project.org/strings - Maximizing Performance with the UnicodeString Storage Model)